tools/bazar/presentation/javascripts/form-edit-template.js

```diff
@@ -563,6 +563,7 @@ var yesWikiTypes = {
   listedatedeb: { type: "date" },
   listedatefin: { type: "date" },
   jour: { type: "date" },
+  map: { type: "map" },
```

Comment:
```suggestion
  map: { type: "map" },
  carte_google: { type: "map" },
```

Il me semble d'après mes tests que ça permet d'avoir :
- création par le côté graphique avec 'map'
- détection que carte_google est un 'map' 
- - renommage automatique de carte_google en map


Qu'en dis-tu ?

Reply:
top!